src/main/scala/algolia/AlgoliaClientConfiguration.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016 Algolia
+ * http://www.algolia.com/
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package algolia
+
+case class AlgoliaClientConfiguration(httpConnectTimeout: Int,
+                                      //httpSocketTimeout in HttpClient
+                                      httpReadTimeout: Int,
+                                      httpRequestTimeout: Int,
+                                      dnsTimeout: Int,
+                                      hostDownTimeout: Int) {}
+
+object AlgoliaClientConfiguration {
+
+  val default = AlgoliaClientConfiguration(
+    httpConnectTimeout = 2 * 1000,
+    httpReadTimeout = 2 * 1000,
+    httpRequestTimeout = 2 * 1000,
+    dnsTimeout = 2 * 100, //200 ms
+    hostDownTimeout = 5 * 60 * 1000
+  )
```

Comment:
Understandable by reading at the code but I would put a comment on top of `default` to explicitly say that all the timeouts are given in ms (instead of a single `//200 ms` in the middle).